src/tutorials.js

```diff
@@ -3,6 +3,14 @@
 const moment = require('moment');
 
 module.exports = Object.freeze([
+  {
+    title: 'Here\'s How to Use Provide/Inject in Vue.js',
+    raw: './tutorials/vue/vue-inject.md',
+    url: '/tutorials/vue/vue-inject',
```

Comment:
Let's always make the tutorial URL like `/vue/inject` as opposed to `/vue/vue-inject`. My understanding is that the former is better for SEO purposes.